docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -0,0 +1,176 @@
+= Using the Data Migration Tool
+:description: This guide explains how to migrate data from Hazelcast 4.x or 5.x clusters to newer 5.x clusters.
+
+[blue]*Hazelcast Enterprise Feature*
+
+{description} Currently, the Data Migration Tool (DMT) supports:
+
+* migrating data only for maps and replicated maps
+* migrating data from Hazelcast 4.x.y clusters to Hazelcast 5.3.2 and newer clusters.
+
+== Before You Begin
+
+* The data migration tool is referred to as DMT throughout this guide.
+* The DMT is available as a separate download on https://viridian.hazelcast.com/[Hazelcast Viridian] and https://hazelcast.com/[Hazelcast web site].
+* The steps in this guide require that you have https://docs.docker.com/get-docker/[Docker] and https://docs.hazelcast.com/clc/latest/install-clc[Hazelcast Command-Line Client (CLC)] installed.
+
+This guide consists of two sets of instructions; one for trying the DMT, other to use it with your existing cluster and maps.
+
+== Trying Out the DMT
+
+The instructions below for trying out the DMT use a map in Hazelcast 4.x.y cluster whose data is migrated to that of Hazelcast {full-version} cluster.
+
+The steps can be summarized as follows:
+
+. Start a source cluster.
+. Create a map and put data into it.
+. Start the target cluster.
+. Start the migration cluster.
+. Migrate map data from the source cluster to the target.
+. Verify the data in the target cluster.
+
+=== Step 1. Start the Source Cluster
+
+Run the following Docker command to start a one-member cluster.
+
+[source,shell]
+----
+$ docker run -p 127.0.0.1:5701:5701 -e HZ_CLUSTERNAME=source hazelcast/hazelcast:4.x.y
```

Comment:
I think it would be better to have a full version here - makes it easier to copy paste, and in the note say it can be any version 4.x.y and 5.x.y